src/gdpr_api.erl

```diff
@@ -59,7 +59,8 @@ process_error({badmatch, {error, ErrorCode}}, ZipFile)
     when ErrorCode =:= eacces orelse ErrorCode =:= erofs ->
         ErrorMessage = "Permission to create file in location '" ++ ZipFile ++ "' denied",
         {file_creation_permission_denied_error, ErrorMessage};
-process_error({badmatch, {error, eisdir}}, ZipFile) ->
+process_error({badmatch, {error, ErrorCode}}, ZipFile)
+    when ErrorCode =:= eisdir orelse ErrorCode =:= eexist ->
```

Comment:
I haven't checked where this is coming from but `eexist` seems to be a different error than being a directory, does that need to return the same error as well or perhaps it can have a different

Reply:
Yeah, you're right, it means that the file already exists. Which is a little weird, because it comes from this test
```
admin_gdpr_access_denied_filename_is_a_directory(Config, Alice) ->
    Filename = "//",
    Res = admin_retrieve_personal_data(escalus_client:username(Alice), escalus_client:server(Alice),
                                       list_to_binary(Filename), Config),
    ?assertEqual(<<"location_is_a_directory_error">>, get_err_code(Res)).
```
but I can try to change it